datasets/bg/jud_declarations/crawler.py

```diff
@@ -0,0 +1,90 @@
+from typing import Dict, Generator, List, Optional
+from lxml.html import HtmlElement
+
+from zavod import Context, helpers as h
+from zavod.logic.pep import categorise, OccupancyStatus
+
+
+def parse_html_table(
+    table: HtmlElement,
+    skiprows: int = 0,
+    headers: Optional[List[str]] = None,
+) -> Generator[Dict[str, HtmlElement], None, None]:
+    for rownum, row in enumerate(table.findall(".//tr")):
+        if rownum < skiprows:
+            continue
+        cells = row.findall("./td")
+        yield {hdr: c for hdr, c in zip(headers, cells)}
+
+
+def crawl_row(context: Context, row: Dict[str, HtmlElement], doc: HtmlElement):
+    doc.make_links_absolute(context.data_url)
+    str_row = h.cells_to_str(row)
+    name = str_row.pop("name")
+    doc_id_date = str_row.pop("doc_id_date")
+    # Skip the header row
+    if name == "Име" and doc_id_date == "Входящ номер":
+        return
+    if len(doc_id_date.split("/")) == 2:
+        _, date = doc_id_date.split("/")
+    else:
+        date = context.lookup_value("doc_id_date", doc_id_date)
+        if date is None:
+            context.log.warning(f"Invalid doc_id_date: {doc_id_date}")
+        return
+    # Link is in the same cell as the name
+    name_link_elem = HtmlElement(row["name"]).find(".//a")
+    declaration_url = name_link_elem.get("href")
+
+    person = context.make("Person")
+    # We want the same person for 2 different years to have the same ID
+    person.id = context.make_id(name)
+    person.add("name", name, lang="bul")
+    person.add("topics", "role.pep")
+    person.add("sourceUrl", declaration_url)
+
+    position = h.make_position(
+        context,
+        name="Judiciary Official",
```

Comment:
as discussed in slack, let's put the position label in here. And if the quality of transliteration by gpt is good enough, let's add that too.